src/Test/Utilities/Portable/Compilation/OperationTreeVerifier.cs

```diff
@@ -42,6 +44,15 @@ public static string GetOperationTree(IOperation operation, int initialIndent =
             return walker._builder.ToString();
         }
 
+        public static void Verify(string expectedOperationTree, string actualOperationTree)
+        {
+            char[] newLineChars = Environment.NewLine.ToCharArray();
+            string actual = actualOperationTree.Trim(newLineChars);
+            expectedOperationTree = expectedOperationTree.Trim(newLineChars);
+            expectedOperationTree = Regex.Replace(expectedOperationTree, "([^\r])\n", "$1" + Environment.NewLine);
```

Comment:
It will trim just `\r\n` from start and end. We need to retain the newline characters within the text so that assert failure displays the actual and expected IOperation tree in a nicely formatted manner in the test window.